Latest keycloak-client failed to compile

### Describe the bug

Because of this change in Keycloak main https://github.com/keycloak/keycloak/commit/018a0802bc36fa627fa94c5d73ec6b175f018eed , there was a compilation failure in latest keycloak-client:

```
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /home/mposolda/IdeaProjects/keycloak-client/admin-client/target/unpacked/org/keycloak/util/JsonSerialization.java:[28,45] package com.fasterxml.jackson.datatype.jsr310 does not exist
[ERROR] /home/mposolda/IdeaProjects/keycloak-client/admin-client/target/unpacked/org/keycloak/util/JsonSerialization.java:[47,35] cannot find symbol
  symbol:   class JavaTimeModule
  location: class org.keycloak.util.JsonSerialization
[INFO] 2 errors 
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for Keycloak Client Libraries 26.0.0-SNAPSHOT:
[INFO] 
[INFO] Keycloak Client Libraries .......................... SUCCESS [  0.632 s]
[INFO] Keycloak Admin REST Client ......................... FAILURE [  2.402 s]
[INFO] Keycloak Admin REST Client ......................... SKIPPED
[INFO] Keycloak Authz: Client API ......................... SKIPPED
[INFO] Keycloak Authz: Policy Enforcer .................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  3.165 s
[INFO] Finished at: 2024-07-18T11:44:48+02:00
[INFO] ------------------------------------------------------------------------
```

### Version

keycloak-client main

### Expected behavior

Project compiles with running `mvn clean install`

### Actual behavior

Project does not compile

### How to Reproduce?

_No response_

### Anything else?

_No response_

--- a/admin-client-jee/pom.xml
+++ b/admin-client-jee/pom.xml
@@ -44,6 +44,10 @@
             <artifactId>jackson-datatype-jdk8</artifactId>
         </dependency>
         <dependency>
+            <groupId>com.fasterxml.jackson.datatype</groupId>
+            <artifactId>jackson-datatype-jsr310</artifactId>
+        </dependency>
+        <dependency>
             <groupId>org.eclipse.microprofile.openapi</groupId>
             <artifactId>microprofile-openapi-api</artifactId>
         </dependency>

--- a/admin-client/pom.xml
+++ b/admin-client/pom.xml
@@ -44,6 +44,10 @@
             <artifactId>jackson-datatype-jdk8</artifactId>
         </dependency>
         <dependency>
+            <groupId>com.fasterxml.jackson.datatype</groupId>
+            <artifactId>jackson-datatype-jsr310</artifactId>
+        </dependency>
+        <dependency>
             <groupId>org.eclipse.microprofile.openapi</groupId>
             <artifactId>microprofile-openapi-api</artifactId>
         </dependency>

--- a/authz-client/pom.xml
+++ b/authz-client/pom.xml
@@ -57,6 +57,10 @@
             <artifactId>jackson-datatype-jdk8</artifactId>
         </dependency>
         <dependency>
+            <groupId>com.fasterxml.jackson.datatype</groupId>
+            <artifactId>jackson-datatype-jsr310</artifactId>
+        </dependency>
+        <dependency>
             <groupId>org.eclipse.microprofile.openapi</groupId>
             <artifactId>microprofile-openapi-api</artifactId>
         </dependency>

--- a/pom.xml
+++ b/pom.xml
@@ -163,6 +163,11 @@
                 <version>${com.fasterxml.jackson.version}</version>
             </dependency>
             <dependency>
+                <groupId>com.fasterxml.jackson.datatype</groupId>
+                <artifactId>jackson-datatype-jsr310</artifactId>
+                <version>${com.fasterxml.jackson.version}</version>
+            </dependency>
+            <dependency>
                 <groupId>org.jboss.logging</groupId>
                 <artifactId>commons-logging-jboss-logging</artifactId>
                 <version>${commons-logging-jboss-logging.version}</version>

